packages/react-percy-config/src/getWebpackSettings.js

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+
+function sanitizeCustomWebpackConfig(config = {}) {
+  // Do not allow users to customize the entry config
+  delete config.entry;
+
+  // Do not allow users to customize the output config
+  delete config.output;
+
+  // Move module.loaders to module.rules so we can properly merge with them
+  if (config.module && config.module.loaders) {
+    config.module.rules = config.module.loaders;
+    delete config.module.loaders;
+  }
+
+  return config;
+}
+
+export default function getWebpackSettings(webpack, packageRoot) {
+  if (!webpack) {
+    const packageJson = require(path.join(packageRoot, 'package.json'));
+
+    // create-react-app
```

Comment:
We continue to support `create-react-app` out of the box with no `percy.config.js` necessary.

Reply:
:animated_happy_dance: